public/forms/validation/meta.ttl

```diff
@@ -116,12 +116,14 @@ ns6:c09902d3-9421-49e9-a2e6-d3ac42abbca2	rdf:type	rdfs:Class ,
 	form:grouping	"MatchEvery"@en ;
 	sh:resultMessage	"Geef een geldig telefoonnummer in."@nl ;
 	ext:customParameter	"defaultCountry"@en .
-<http://lblod.data.gift/concepts/60873518-5423-4121-ba6a-9635b18242a0>	rdf:type	rdfs:Class ,
```

Comment:
These need to be the constraint class uris I think, I only adjusted it for the MaxLength one for now.